### PR TITLE
Fix #2874 part 2 - Add AddLocalization() overload for MVC localization.

### LINKED ISSUE #2874: Add IMvcBuilder overloads for localization and XML formatters
See here: https://github.com/aspnet/Mvc/issues/2872#issuecomment-125750316

We don't have an `IMvcBuilder` startup experience for these pieces.

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

**Comment by kirthik** [comment]
:shipit:

File: src/Microsoft.AspNet.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    /// </summary>
+    public static class MvcLocalizationMvcBuilderExtensions
+    {
+        /// <summary>
+        /// Adds MVC localization to the application.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/>.</param>
+        /// <returns>The <see cref="IMvcBuilder"/>.</returns>
+        /// <remarks>
+        /// Adding loca
```
Comment (dougbu) [inline src/Microsoft.AspNet.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs]: Might be missing something here. Do you mean "loca" as a slang term, Spanish for crazy, the plural for locus, ...

Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs]: heh. Forgot to finish typing, it's going to say "Adding localization also adds view support by calling AddViews(...)."

**Comment by rynowak** [comment]
updated

File: src/Microsoft.AspNet.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs (hunk 1 above)
Comment (dougbu) [inline src/Microsoft.AspNet.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs]: Sounds good. Should have similar `<remarks>` on the other overload too.

**Comment by dougbu** [comment]
:shipit:

**Pull request closed without merging**